addons/website/static/src/scss/website.scss

```diff
@@ -2057,8 +2057,9 @@ input[value*="data-oe-translation-source-sha"] {
     visibility: visible;
 }
 .o_wanim_overflow_xy_hidden {
-    overflow-x: hidden !important;
-
+    > body {
+        overflow-x: hidden !important;
+    }
```

Comment:
@bvr-odoo This needs comments in the code, it's not obvious

Reply:
> @bvr-odoo This needs comments in the code, it's not obvious

@qsm-odoo I added the comment